frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBelowAction.java

```diff
@@ -209,7 +209,9 @@ public void onFinished() {
     return Tuple.of(options.getName(), preSlideInRunnable);
   }
 
-  private static @Nullable GitRemoteBranch getGitRemoteBranch(Project project, GitRepository gitRepository, String branchName) {
+  private static @Nullable GitRemoteBranch getGitRemoteBranch(Project project,
+      @NonNull GitRepository gitRepository,
```

Comment:
Redundant `@NonNull`?

Reply:
We are calling `gitRepository.getRemotes` in the body of the method.

Reply:
Hmmm okay but doesn't it compile without `@NonNull`? Checker assumes everything to be non-null by default 🤔

Reply:
It compiled, so I removed the tag.